### PR TITLE
Adds content for devsprint page

### LINKED ISSUE #99: Add DevSprint Page
Use content from [wiki](https://github.com/pythonindia/inpycon2024/wiki/PyCon-India-2024-Devsprint) and [DevSprint CFP](https://in.pycon.org/cfp/devsprint-2024/proposals/) Page.

### PR DESCRIPTION
**Pull request by SaptakS**
Fixes #99 

I have not added the projects from Junction and instead provided a link to it since I guess people might still submit more devsprint proposals?

cc @anistark @bhavin192 for content review

### CODE REVIEW

**Review by farhaanbukhsh — APPROVED** [review]
LGTM

File: src/devsprint.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## When and Where?
+
+Devsprints for this edition of PyCon India will also be conducted Google Office, Bengaluru. The exact location and format on how it will be conducted will be communicated to the Mentors/Attendees close to the Sprints.
+
+## Submit a Project
+If you are a contributor or have been involved in the maintenance of any Open Source project, please come forward and add your project. This project can be any interesting Open source project which is based on Python. Ideally, you should have a history with the project through previous code commits. Your responsibility will be to help your team set up their development environment, list out beginner-friendly bugs, and guide them on any problems they face during the event.
+
+A list of submitted devsprint proposal can be seen [here](https://in.pycon.org/cfp/devsprint-2024/proposals/).
```
Comment (bhavin192) [inline src/devsprint.md]:
```suggestion
A list of submitted Devsprint proposals can be seen here: <https://in.pycon.org/cfp/devsprint-2024/proposals/>
```

We can consider leaving the full link, I usually prefer either links or more text hyperlinked than just words like here, this etc. 
Gives better idea to reader where the link will take them.

File: src/devsprint.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### How to add a Project?
+
+You can propose the project using [junction](https://in.pycon.org/cfp/devsprint-2024/proposals/).
```
Comment (bhavin192) [inline src/devsprint.md]:
```suggestion
You can propose the project using [CFP platform - Junction](https://in.pycon.org/cfp/devsprint-2024/proposals/).
```

How about CFP platform Junction, reader might not know what's Junction.

**Comment by bhavin192** [comment]
@farhaanbukhsh how we want to write it?
1. Devsprint
2. DevSprint

On junction most of the text writes it as Devsprint or devsprint but Junction event name has DevSprint. 
Consistency would be good to have for now, otherwise ignore this comment.

**Comment by farhaanbukhsh** [comment]
> @farhaanbukhsh how we want to write it?
> 
> 1. Devsprint
> 2. DevSprint
> 
> On junction most of the text writes it as Devsprint or devsprint but Junction event name has DevSprint. Consistency would be good to have for now, otherwise ignore this comment.

I have generally followed DevSprint but either way it is okay.

**Comment by farhaanbukhsh** [comment]
@SaptakS @bhansa Here is a link to the participant form:

https://forms.gle/LSZeuHfxkfqh2TNx8

It can be a good addition to the page

**Comment by SaptakS** [comment]
@bhavin192 @farhaanbukhsh ready for re-review.

**Review by anistark — APPROVED** [review]
Works for now. 

Better to add all projects that participated in this sprint.

**Pull request merged**